test/test_helper.rb

```diff
@@ -1,6 +1,3 @@
-require 'coveralls'
-Coveralls.wear!
-
 $:.unshift(File.expand_path(File.dirname(__FILE__) + '../../lib/'))
```

Comment:
Oh, I remember now! It's because `.simplecov` has the 'require' commands as well. I'm going to try to simplify this....